karma.conf.js

```diff
@@ -58,9 +58,12 @@ module.exports = function(config) {
     }
 
     let port = 0;
+    let mochaTimeout = 2000;
     if (config.debug) {
         port = 9876;
         webpackConfig.devtool = 'inline-source-map';
+        // When debuging in the browser, do not timeout
+        mochaTimeout = 1000000;
```

Comment:
Why using an arbitrary high value when using `0` would simply disable the timeout altogether ?

Reply:
I didn't know, you're right.